Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java

```diff
@@ -531,32 +524,7 @@ private DocStructInterface getDocStruct(AdditionalField field) throws Preference
         return docStruct;
     }
 
-    /**
-     * Get together authors' names.
-     *
-     * @param persons
-     *            list of persons
-     * @return authors' names as String
-     */
-    protected String getAuthors(List<PersonInterface> persons) {
```

Comment:
why did you remove this?

Reply:
Because there is no dedicated `Person` object any more in the future. Complex object descriptions, such as persons, can and have to be done using meta-data groups.